src/util/psbt/error.rs

```diff
@@ -127,7 +132,8 @@ impl std::error::Error for Error {
             | NonStandardSighashType(_)
             | InvalidPreimageHashPair{ .. }
             | CombineInconsistentKeySources(_)
-            | ConsensusEncoding => None,
+            | ConsensusEncoding
+            | IncompleteTapTree(_) => None,
```

Comment:
The inner `IncompleteTapTree` implements `error::Error` so this should return `Some(e)` instead of `None`, same as for `HashParse`.

Reply:
s/should/must/